c2corg_ui/templates/article/edit.html

```diff
@@ -0,0 +1,143 @@
+                <p ng-message="minlength" class="label label-danger" style="{{ (editForm.title.$invalid && editForm.title.$touched) ? 'display:inline' : '' }}" translate>Title must have at least 3 characters.</p>
+              </div>
+            </div>
+
+            ## LANGUAGE
+            <div id="lang-group" class="form-group" ng-class="{ 'has-error': editForm.lang.$touched && editForm.lang.$invalid, 'has-success': editForm.lang.$valid}">
+              <label><span translate>lang</span> *</label>
+              % if article_lang:
+                <input disabled class="form-control" value="{{mainCtrl.translate('${article_lang}')}}">
+              % else:
+              <select name="lang" ng-options="lang as mainCtrl.translate(lang) for lang in ['${"','".join(default_langs) |n}'] | translate" ng-model="article.locales[0].lang" class="form-control" required><option value=""></option></select>
+              % endif
+              <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="article.locales[0].lang"></span>
+              <span class="glyphicon glyphicon-warning-sign form-control-feedback" ng-show="editForm.lang.$touched && editForm.lang.$invalid"></span>
+              <div class="help-block" ng-messages="editForm.lang.$error">
+                <p ng-message="required" class="label label-danger" style="{{ (editForm.lang.$invalid && editForm.lang.$touched) ? 'display:inline' : '' }}" translate>This field is required.</p>
+              </div>
+            </div>
+
+            <section class="section">
+
+              ## ACTIVITIES
+              <div class="form-group data full" id="route-activities" ng-init="activities = ${activities}"
+                   ng-class="{ 'has-error': editForm.activities.$touched && editForm.activities.$invalid, 'has-success': editForm.activities.$valid }">
+                <label><span translate>activities</span> *</label>
+                <div class="route-activities">
+                  <div ng-repeat="activity in activities" class="activity">
+                    <div>
+                      <div class="icon-{{activity}}" class="icon-" ng-click="editCtrl.pushToArray(article, 'activities', activity, $event)"
+                           ng-class="{'activity-selected' : article.activities.indexOf(activity) > -1}">
+                      </div>
+                      <p>{{activity | translate}}</p>
+                    </div>
+                  </div>
+                </div>
+              </div>
+
+            </section>
+
+            <div class="form-group half half-group">
+
+              <div ng-class="{ 'has-success': article.article_type}">
+                <label><span translate>article_type</span> *</label>
+                <select class="form-control" ng-model="article.article_type" ng-options="rat as mainCtrl.translate(rat) for rat in ${article_types}"><option value=""></option></select>
```

Comment:
OK when creating a new article.
But when editing a collaborative article, only moderators should be allowed to change the ``article_type`` to ``personal``.
On the other hand, no problem to change a ``personal`` article to ``collaborative``.